python/ray/_private/gcs_aio_client.py

```diff
@@ -205,9 +205,16 @@ async def internal_kv_keys(
     async def get_all_job_info(
         self,
         job_or_submission_id: Optional[str] = None,
```

Comment:
nit: since there are more parameters, let's force them to be keyword arguments by doing

```
get_all_job_info(self, *, job_or_submission_id, ...)
```